src/apps/frontend/constants/routes.ts

```diff
@@ -2,6 +2,8 @@ const constants = {
   ABOUT: '/about',
   DASHBOARD: '/',
   LOGIN: '/login',
+  OTP: '/otp',
+  PHONELOGIN: '/phone-login',
```

Comment:
Poor naming convention. Since you have two words here, it should be `PHONE_LOGIN`. In general, the url for `/otp` and `/phone-login` are quite poor as well.

Rather, the route should be

`/sign_up?auth_mode=otp`
`/login?auth_mode=otp`

And those pages should accordingly render appropriate forms.